charts/super-agent/values.yaml

```
@@ -94,71 +94,84 @@ super-agent-deployment:
   # @default -- `false`
   nrStaging:
 
-  # -- (bool) Changes default endpoint to point to EU backend.
-  # @default -- `false`
-  euEndpoints:
-
   fedramp:
     # -- (bool) TODO: Enables FedRAMP. Can be configured also with `global.fedramp.enabled`
     # @default -- `false`
     enabled:
 
-  # -- (bool) TODO: Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog`
+  # -- (bool) Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog`
   # @default -- `false`
   verboseLog:
 
   # -- Enable the cleanup of super-agent managed resources when the chart is uninstalled.
-  # If disabled, agents and / or agent configurations managed by the super-agent will not be deleted when the chart is uninstalled.
+  # If disabled, agents and/or agent configurations managed by the super-agent will not be deleted when the chart is uninstalled.
   cleanupManagedResources: true
 
   config:
-    auth:
-      enabled: false
-      organizationId:
-      userApiKey:
-      secret:
-        create: true
-
     # -- Configuration for the Super Agent.
     # @default -- See `values.yaml`
     superAgent:
       # -- Set if the configMap is going to be created by this chart or the user will provide its own.
       create: true
-      # -- Here you can set New Relic' Super Agent configuration.
-      # @default -- See `values.yaml` for examples
-      content:
-        agents:
-          open-telemetry:
-            agent_type: newrelic/io.opentelemetry.collector:0.2.0
-
-        # opamp:
-        #   # TODO The endpoint should be set automatically based on the licenseKey and on the nrStaging option if opamp.enable=true
-        #   endpoint: https://opamp.service.newrelic.com/v1/opamp
-        #   endpoint: https://opamp.service.eu.newrelic.com/v1/opamp
-
-        # -- This option enables a status server that can be useful for troubleshooting.
-        # -- Port-forward it `$ kubectl port-forward pod/{pod-name} 51200:51200`
-        # -- And query it as `$ curl localhost:51200/status`
-        server:
-          enabled: true
-        #   port: 51200
-
-
-    # -- Values that the fleet is going to have in the deployment.
-    # @default -- See `values.yaml` for examples
+      # -- It you are a PowerUser, you can use this to override the configuration that has been created automatically by the chart.
```

what is a PowerUser ? this might be misleading

---

I wanted to keep the possibility to create a config directly from the values that is passed to the super agent.

That is something that only somebody that know what is doing should use that value. I called it power user because it is __kind of a hacker__ let's say.

I am open to any suggestion on which would you like to have in that comment.

---

make sense, but i would just mention what it does and maybe add some warns. 
```suggestion
      # -- Overrides the configuration that has been created automatically by the chart.
```